nautobot/docs/plugins/development.md

```diff
@@ -141,10 +156,32 @@ All required settings must be configured by the user. If a configuration paramet
 
 ### Install the Plugin for Development
 
-To ease development, it is recommended to go ahead and install the plugin at this point using setuptools' `develop` mode. This will create symbolic links within your Python environment to the plugin development directory. Call `setup.py` from the plugin's root directory with the `develop` argument (instead of `install`):
+The plugin needs to be installed into the same python environment where Nautobot is, so that we can get access to `nautobot-server` command, and also so that the nautobot-server is aware of the new plugin.  
+
+If you installed Nautobot using poetry, then go to the Nautobot repo clone and run `poetry shell` there.  
+
+Otherwise, manually activate nautobot using `source /opt/nautobot/bin/activate`.
```

Comment:
```suggestion
Otherwise if you're using the Docker workflow, manually activate the Nautobot environment using `source /opt/nautobot/bin/activate`.
```